online-boutique-tenant/kustomize/cluster-role.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: online-boutique-role
+rules:
+- apiGroups:
+  - velero.io
+  resources:
+  - schedules
+  verbs:
+  - create
+  - patch
+  - update
+  - delete
+- apiGroups:
+  - sloth.slok.dev
+  resources:
+  - prometheusservicelevels
+  verbs:
+  - create
+  - patch
+  - update
+  - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Why allow online-boutique tenant to create resources at the cluster wide level? 
See https://github.com/ccl-consulting/k8saas/pull/699#discussion_r2005923258

These roles should be added at application level and aggregated to the view and admin roles

Reply:
Tracked at #762 
Implementation done at https://github.com/ccl-consulting/k8saas/pull/764
Testing in progress